apps/consent/BUCK

```diff
@@ -43,11 +43,12 @@ filegroup(
     "public/**",
     "services/**",
     "components/**",
-    "env.ts",
     "tailwind.config.ts",
     "postcss.config.js",
     "next.config.js",
     "tsconfig.json",
+    "*.ts", # env.ts / instrumentation.ts
+    "instrumentation.node.ts"
```

Comment:
if *.ts is working then `instrumentation.node.ts` is redundant - or why does the comment say `instrumentation.ts`?

Reply:
Okay I see there are 2 files.